src/enhancers/calendar-enhancers.js

```diff
@@ -0,0 +1,17 @@
+const moment = require('moment');
+
+export const __calculateDayByKey = (code, focusedDay) => {
```

Comment:
I think it is a good idea to pull this out to make things testable. Would it make more sense to pass in focusedDay as a moment object so it can be pure, or will that make the component harder to test?

Reply:
also, what's the thinking behind the double underscore function name prefix?

Reply:
it doesn't have to be double underscore, but I think something to show that it's coming from a utils file? this way when you see it in the code base you know it's not a class method (you also will probably know because there will be no `this.` when invoking it... 

could be overkill